library/src/main/java/com/mikepenz/materialize/color/Material.java

```diff
@@ -48,5 +48,759 @@ public int getAsResource() {
+            return color;
+        }
+
+        @Override
+        public int getAsColor() {
+            return Color.parseColor(color);
+        }
+
+        @Override
+        public int getAsResource() {
+            return resource;
+        }
+    }
+
+    public enum Grey implements IColor {
+
+        _50("#FAFAFA", R.color.md_grey_50),
+        _100("#F5F5F5", R.color.md_grey_100),
+        _200("#EEEEEE", R.color.md_grey_200),
+        _300("#E0E0E0", R.color.md_grey_300),
+        _400("#BDBDBD", R.color.md_grey_400),
+        _500("#9E9E9E", R.color.md_grey_500),
+        _600("#757575", R.color.md_grey_600),
+        _700("#616161", R.color.md_grey_700),
+        _800("#424242", R.color.md_grey_800),
+        _900("#212121", R.color.md_grey_900);
+
+        String color;
+        int resource;
+
+        Grey(String color, int resource) {
+            this.color = color;
+            this.resource = resource;
+        }
+
+        @Override
+        public String getAsString() {
+            return color;
+        }
+
+        @Override
+        public int getAsColor() {
+            return Color.parseColor(color);
+        }
+
+        @Override
+        public int getAsResource() {
+            return resource;
+        }
+    }
+
+    public enum BlueGrey implements IColor {
+
+        _50("#ECEFF1", R.color.md_blue_grey_50),
+        _100("#CFD8DC", R.color.md_blue_grey_100),
+        _200("#B0BEC5", R.color.md_blue_grey_200),
+        _300("#90A4AE", R.color.md_blue_grey_300),
+        _400("#78909C", R.color.md_blue_grey_400),
+        _500("#607D8B", R.color.md_blue_grey_500),
+        _600("#546E7A", R.color.md_blue_grey_600),
+        _700("#455A64", R.color.md_blue_grey_700),
+        _800("#37474F", R.color.md_blue_grey_800),
+        _900("#263238", R.color.md_blue_grey_900);
+
+        String color;
+        int resource;
+
+        BlueGrey(String color, int resource) {
+            this.color = color;
+            this.resource = resource;
+        }
+
+        @Override
+        public String getAsString() {
+            return color;
+        }
+
+        @Override
+        public int getAsColor() {
+            return Color.parseColor(color);
+        }
+
+        @Override
+        public int getAsResource() {
+            return resource;
+        }
+    }
+
+    public enum Monochrome implements IColor {
+
+        Black("#000000", R.color.md_black_1000),
+        White("#FFFFFF", R.color.md_white_1000);
+
+        String color;
+        int resource;
+
+        Monochrome(String color, int resource) {
+            this.color = color;
+            this.resource = resource;
+        }
+
+        @Override
+        public String getAsString() {
+            return color;
+        }
+
+        @Override
+        public int getAsColor() {
+            return Color.parseColor(color);
+        }
+
+        @Override
+        public int getAsResource() {
+            return resource;
+        }
+    }
+
 }
```

Comment:
Here for white: 
https://github.com/mikepenz/MaterialDrawer/blob/develop/library/src/main/res/values/colors.xml#L51

And here for black:
https://github.com/mikepenz/MaterialDrawer/blob/develop/library/src/main/res/values/colors.xml#L12